futures-channel/src/lib.rs

```diff
@@ -0,0 +1,66 @@
+//! Asynchronous channels
+//!
+//! This crate provides channels which can be used to communicate between futures.
+
+#![deny(missing_docs, missing_debug_implementations)]
+#![doc(html_root_url = "https://docs.rs/futures/0.2")]
+
+#[macro_use]
+extern crate futures_core;
+
+if_std! {
```

Comment:
I think since this whole crate requires std it's fine to not bother with `if_std!` here